src/pages/docs/getting-started/laravel.mdx

```diff
@@ -6,40 +6,15 @@ meta_keywords: "Pub/Sub Laravel 12, Laravel PubSub, Ably Laravel SDK, publish La
 
 This guide will get you started with Ably Pub/Sub in Laravel 12.
 
-It will take you through the following steps:
-
-* Create a client and establish a connection to Ably with:
-  * The Laravel Broadcaster an independent service provider library for [Laravel](https://laravel.com/) using [ably-php](https://github.com/ably/ably-php)
-  * [Echo](https://github.com/ably-forks/laravel-echo), a Laravel Echo fork that adds native Ably integration for Laravel's realtime broadcasting features.
-* Create an event to broadcast (publish) messages from the backend.
-* Set up a frontend client to receive (subscribe) and publish messages to a channel.
+It will take you through creating a client and establishing a connection to Ably using the [Laravel Broadcaster](https://github.com/ably/laravel-broadcaster) service provider and [Echo](https://github.com/ably-forks/laravel-echo) (a Laravel Echo fork with native Ably integration), then creating an event to broadcast messages from the backend and setting up a frontend client to receive and publish messages to a channel.
```

Comment:
`It will take you through...` We should align this with the other guides, which is more along the lines of `You'll learn...`. This is also quite a long sentence that I think it would be worth breaking up.